frontend/src/scenes/surveys/SurveyTemplates.scss

```diff
@@ -15,8 +15,6 @@
     }
 
     .SurveyTemplate {
-        -ms-transform: scale(0.8, 0.8); /* IE 9 */
-        -webkit-transform: scale(0.8, 0.8); /* Safari */
```

Comment:
Do we not support these browsers?

Reply:
The app definitely doesn't support browsers requiring these prefixes. And if we did, we should only rely on autoprefixer, not remember this manually.